pkgs/development/python-modules/dictdiffer/default.nix

```
@@ -0,0 +1,49 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, pytest-runner
+, setuptools-scm
+, tox
+, sphinx
+, check-manifest
+, mock
+, pytest
+, pytest-cov
+, pytest-isort
+, pytest-pycodestyle
+, pytest-pydocstyle
+}:
+
+buildPythonPackage rec {
+  pname = "dictdiffer";
+  version = "0.9.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0y3mk74qm2q9hsm37892i1wzn8bbdrvbs4nmnvqwq4z6pxgwzfhp";
+  };
+
+  checkInputs = [
+    check-manifest
+    mock
+    pytest
+    pytest-cov
+    pytest-isort
+    pytest-pycodestyle
+    pytest-pydocstyle
+    pytest-runner
+    sphinx
+    tox
```

```suggestion
    sphinx
```
We don't run tests through tox because that is causing sometimes issues.
pytest-runner is useless in nixpkgs.
checkPhase should never run linter like isort, pycodestyle or pydocstyle to make upgrades on those linters less painfull.
cov report is never generated because it slows down the tests, it can't be submmited which makes it useless.

---

So should I disable all tests then?

---

This was added in another PR